README.md

```diff
@@ -98,7 +98,7 @@ api.uploadData(uploadDataPath).then(function(metadata) {
 });
 ```
 
-List uploaded data:
+List all uploaded data:
```

Comment:
Again, I find the `all` to be redundant.

Reply:
yeah, that's true.